src/charm.py

```diff
@@ -723,6 +775,42 @@ def _on_run_migration_action(self, event: ActionEvent) -> None:
             return
         event.log("Successfully migrated the database.")
 
+    def _on_loki_consumer_endpoint_joined(self, event: LogProxyEndpointJoined) -> None:
+        logger.info("Loki Push API endpoint joined")
+
+    def _on_loki_consumer_endpoint_departed(self, event: LogProxyEndpointDeparted) -> None:
```

Comment:
Similar to @sed-i 's comment above, do we need to handle this event?

Reply:
Not really, I left it in for logging purpuses. It was useful while debugging. I'll take it out then.